core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -561,6 +561,11 @@ private[deploy] class Master(
     state = RecoveryState.ALIVE
     schedule()
     logInfo("Recovery complete - resuming operations!")
+	} catch {
```

Comment:
Thank you very much, I've changed the commit,  you can see  if there are any other problems.